elements/pfe-tabs/demo/index.html

```diff
@@ -173,8 +173,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Vertical earth</h2>
-      <pfe-tabs vertical pfe-variant="earth">
+      <h2>Style: Vertical earth </h2>
+      <pfe-tabs vertical pfe-align="center" >
```

Comment:
Does this need the earth variant added back to it?

Reply:
gah, thank you for catching that. Twas unintentional, and has been resurrected.